packages/wonder-blocks-timing/src/hooks/internal/use-simple-interval.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {useEffect} from "react";
+
+import {useUpdatingRef} from "./use-updating-ref.js";
+
+/**
+ * A simple hook for using `setInterval`.
+ *
+ * @param action called every `intervalMs` when `isSet` is true
+ * @param intervalMs the duration between calls to `action`
+ * @param isSet whether or not the interval is active
+ */
+export function useSimpleInterval(
+    action: () => mixed,
+    intervalMs: number,
+    isSet: boolean,
+) {
+    const actionRef = useUpdatingRef(action);
```

Comment:
I'm curious as to what benefits we get from `useUpdatingRef`? It feels a bit counter-intuitive to me. 

My first thought when reading this was wondering whether or not `useCallback` could be used here for the same purpose

Reply:
Using a ref avoids recreating the interval.